### PR TITLE
style: fix formatting

### PR DESCRIPTION
**Pull request by evilrobot-01**
Quick fix to correct formatting.

### CODE REVIEW

**Review by oraclown — APPROVED** [review]
Looks fine. I'm getting some warnings (looks like GH beta feature). Not sure if those need to be addressed elsewhere. I'll just link 'em in case they don't show up for you:
```
this function has too many arguments (9/7)
warning: this function has too many arguments (9/7)
   --> src/lib.rs:561:12
   
this `else { if .. }` block can be collapsed
warning: this `else { if .. }` block can be collapsed
    --> src/lib.rs:1359:12

unused variable: `address`
warning: unused variable: `address`
    --> src/lib.rs:1453:4

an implementation of `From` is preferred since it gives you `Into<_>` for free where the reverse isn't true
warning: an implementation of `From` is preferred since it gives you `Into<_>` for free where the reverse isn't true
  --> src/xcm/mod.rs:84:1

using `clone` on type `VoteResult` which implements the `Copy` trait
warning: using `clone` on type `VoteResult` which implements the `Copy` trait

very complex type used. Consider factoring parts into `type` definitions
warning: very complex type used. Consider factoring parts into `type` definitions
   --> src/impls.rs:244:7

associated function `tally_votes` is never used
warning: associated function `tally_votes` is never used
   --> src/impls.rs:938:16

function `vote` is never used
warning: function `vote` is never used
  --> src/contracts/governance.rs:25:15
```

**Comment by evilrobot-01** [comment]
Thanks, will address before merge.

**Comment by evilrobot-01** [comment]
Have fixed one issue, most of the rest relate to work that is still in progress and should disappear soon.

I will merge when the current CI completes successfully and create an issue to resolve the remaining clippy warnings then.

**Pull request merged**
